package.json

```diff
@@ -210,7 +215,10 @@
 		"universal-analytics": "^0.5.1",
 		"urlsafe-base64": "^1.0.0",
 		"uuid": "^8.3.0",
-		"winston": "^3.8.2"
+		"winston": "^3.8.2",
+		"y-mongodb-provider": "^0.1.7",
```

Comment:
https://www.npmjs.com/package/y-mongodb-provider it not a good libary. It they needed, can it done by us self.
Please remove/replace it in a follow up.